test/application/widgets/theme/highlightable_text_test.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:memo/application/widgets/theme/highlightable_text.dart';
+
+import '../../../utils/widget_pump.dart';
+
+void main() {
```

Comment:
Could add a test for scenarios where fakeHighlight is "empty" text, null or highlight not present in the actual text

Reply:
Could add a test for case-insentivity

Reply:
Nice suggestions